app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java

```diff
@@ -70,7 +81,61 @@ public void onClick(View v) {
         });
         return rootview;
     }
+    public List<EnhancementOptionsEntity> getEnhancementOptions() {
+        mTextEnhancementOptionsEntityArrayList.clear();
 
+        mTextEnhancementOptionsEntityArrayList.add(
+                new EnhancementOptionsEntity(getResources().getDrawable(R.drawable.ic_font_black_24dp),
+                        getResources().getString(R.string.edit_font_size)));
+        return mTextEnhancementOptionsEntityArrayList;
+    }
+    private void showEnhancementOptions() {
```

Comment:
Leave lines between the function & add small comment to every newly added function